www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf

```diff
@@ -8,19 +8,19 @@ virus_scan.ScanFileTypes {{ OPNsense.cicap.antivirus.scanfiletypes.replace(',',
 {% endif %}
 
 {% if helpers.exists('OPNsense.cicap.antivirus.sendpercentdata') and OPNsense.cicap.antivirus.sendpercentdata != '' %}
-virus_scan.SendPercentData 5
+virus_scan.SendPercentData {{ OPNsense.cicap.antivirus.sendpercentdata }}
 {% endif %}
 {% if helpers.exists('OPNsense.cicap.antivirus.startsendpercentdataafter') and OPNsense.cicap.antivirus.startsendpercentdataafter != '' %}
-virus_scan.StartSendPercentDataAfter 2M
+virus_scan.StartSendPercentDataAfter {{ OPNsense.cicap.antivirus.startsendpercentdataafter }}
 {% endif %}
 {% if helpers.exists('OPNsense.cicap.antivirus.allow204responses') and OPNsense.cicap.antivirus.allow204responses != '' %}
-virus_scan.Allow204Responces on
+virus_scan.Allow204Responces {{ OPNsense.cicap.antivirus.allow204responses }}
```

Comment:
@mimugmail please be careful - on/off switches of checkboxes are 1/0 - you may break the config here